pages/common/funzip.md

```diff
@@ -0,0 +1,12 @@
+# funzip
+
+> Print the content of the first (non-directory) member in an archive without extraction.
+> More information: <https://manned.org/funzip>.
+
+- Print the content of the first member in a `.zip` archive:
+
+`funzip {{path/to/archive.zip}}`
```

Comment:
add an example description for Gzip files too.

Reply:
@navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example

Reply:
> @navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example

You can update it right here on GitHub in Files changed tab.

Reply:
> > @navarroaxel , I am not able find this file on my repo as the branch was deleted from my end. Please can you approve this PR. I will submit a new PR adding the gzip example
> 
> You can update it right here on GitHub in Files changed tab.

@kbdharun The edit option is not clickable. Cannot edit or delete the file

Reply:
Same here, I guess you could create a new PR to add this page. In the new PRs description add closes #this-pr-number.

Reply:
sure, that make sense

Reply:
@kbdharun , I have created a new PR #9236. We can close this PR